src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: $card-width;
+  padding: 16px;
+  background: $color-main-background;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
```

Comment:
It's better to group similar style values(e.g. font styles, position properties).